### ISSUE TITLE
Hiding title-description-cover image

### ISSUE THREAD

**Issue by ugurcany** [royalfig/smart#73]
Is there a way to hide the area between navbar and the content on the home page? It takes too much space and doesn't serve for any purpose.

**Comment by royalfig** [comment]
I don't agree with your opinion, but you can adjust the size by changing the height on the `sm-hero` class.

For example, you could add the following to your Ghost site's code injection header:
```css
<style>
.sm-hero {
    height: 30vh;
}
</style>
```

To remove the hero section entirely, it gets a little more tricky, but it's possible. You can fork the theme and make the changes you want. 😄
